common/config/rush/pnpm-lock.yaml

```diff
@@ -2524,20 +2512,20 @@ packages:
       node: '>=0.8'
     resolution:
       integrity: sha512-nlLsUzgm1kfLXSXfRZMc1KLAugd4hqJHDTvc2hDIwS3mZAfMEuMbc03SujMF+GEcpaX/qboeycw6iO8JwVv2+g==
+  /tr46/0.0.3:
+    dev: false
+    resolution:
+      integrity: sha1-gYT9NH2snNwYWZLzpmIuFLnZq2o=
   /tr46/1.0.1:
     dependencies:
```

Comment:
Interesting that we get v0.0.3 and v1.0.1 here.

Reply:
found the issue, updating `node-fetch` to the latest remove the need for 0.0.3 through many nested dependencies